app/services/client.py

```diff
@@ -0,0 +1,22 @@
+import requests
+from dotenv import load_dotenv
+from os import getenv
+
+
+load_dotenv()
+
+class WordsClient:
+    def __init__(self, base_url: str, apiKey: str) -> None:
+        self.base_url = base_url
+        self.apiKey = apiKey
+
+
+    def RandomWord(self):
+        r = requests.get(f"{self.base_url}/getRandom", headers= {"X-RapidAPI-Key": self.apiKey, "X-RapidAPI-Host": "random-words5.p.rapidapi.com"})
+        return r.text
+    
+    def RandomWords(self):
+        r = requests.get(f"{self.base_url}/getMultipleRandom", params={"count": "5"},headers= {"X-RapidAPI-Key": self.apiKey, "X-RapidAPI-Host": "random-words5.p.rapidapi.com"})
+        return r.text
+
+APIclient = WordsClient(base_url="https://random-words5.p.rapidapi.com", apiKey=getenv("APIKEY"))
```

Comment:
```suggestion
words_client = WordsClient(base_url=getenv("words_base_url"), api_key=getenv("APIKEY"))
```
APIclient is just a regular variable so it should be with snake_case

words_client = WordsClient()

also export the base url to env as well